src/internal/connector/exchange/api/contacts.go

```diff
@@ -55,6 +55,41 @@ func (c Contacts) CreateContactFolder(
 	return mdl, nil
 }
 
+// GetContactFolderByDisplayName fetches contact folder which has the displayName of
+// folderName. This is only done at root level.
+func (c Contacts) GetContactFolderByDisplayName(
```

Comment:
A design we're moving towards, but haven't reached yet, is the ability to use the api iterations (mail, contacts, events) as common interfaces throughout connector/exchange.  In order to do that, we need to keep common func naming in mind.  I'd suggest the following here (just swapping "contactFolder" for "container").  This will make it practical and easy to, later on, establish an interface like `ContainerByNamer { GetContainerByDisplayName(...) }`

```suggestion
func (c Contacts) GetContainerByDisplayName(
```

Reply:
Good point! I'll make the change.